servers/src/test/java/tachyon/underfs/UnderFileSystemCluster.java

```diff
@@ -105,7 +105,7 @@ public static UnderFileSystemCluster getUnderFilesystemCluster(String baseDir,
    */
   public static boolean readEOFReturnsNegative() {
     // TODO Should be dynamically determined - may need additional method on UnderFileSystem
-    return (null != sUfsClz) && !(sUfsClz.equals("tachyon.LocalUnderFileSystem"));
+    return null != sUfsClz && sUfsClz.equals("tachyon.underfs.hdfs.LocalMiniDFSCluster");
```

Comment:
Can you update the comment for this method to point to the right test? I think FileOutStreamTest was moved/renamed.

And, how does the name "tachyon.underfs.hdfs.LocalMiniDFSCluster" get used? (how did it work before?)